### PR TITLE
Main REACT functionalities

### PR DESCRIPTION
**Pull request by mrzengel**
Closes #14, #7

### CODE REVIEW

File: zenodo_jupyterlab/labextension/package.json [hunk 1]
```diff
@@ -7,10 +7,14 @@
...
-  "homepage": "",
+  "homepage": "https://github.com/vre-hub/zenodo-jupyterlab-extension",
   "bugs": {
     "url": "/issues"
   },
+  "private": true,
+  "workspaces": [
+    "zenodo_jupyterlab"
+  ],
   "license": "BSD-3-Clause",
```
Comment (garciagenrique) [inline zenodo_jupyterlab/labextension/package.json:18]: To be changed on the next PR

**Review by garciagenrique — APPROVED** [review]

**Review by garciagenrique — COMMENTED** [review]
Why there are two files inside `zenodo_jupyterlab/labextension` ?
`build_log.json` and a second `package.json` ?

Could it be that you run the building stage inside this directory ?

File: zenodo_jupyterlab/labextension/package.json (hunk 1 above)
Comment (mrzengel) [reply zenodo_jupyterlab/labextension/package.json:18]: Yes. I was planning on pulling from main to dev after we approve both pull requests

**Comment by garciagenrique** [comment]
> I looked into it and those files also exist on the main branch. Today was the only day I messed around in that folder, so I'm not sure where they came from.

Could you please gitignore then for the moment ? I don't thing we'd need them but just in case

**Review by garciagenrique — APPROVED** [review]
lgtm.
CI to be fixed

**Pull request merged**